package.json

```diff
@@ -64,8 +64,9 @@
         "*.less": [
             "stylelint --fix"
         ],
-        "*.{js,ts,html, svg,yml,md,less,json}": [
-            "prettier --write"
+        "*.{js,ts,html,svg,yml,md,less,json}": [
+            "prettier --write",
+            "cspell"
```

Comment:
`npm run cspell` ? 🤔

Reply:
```
> cspell
 1/10 ./scripts/check-dist.ts 1659.55ms
 2/10 ./scripts/combine-playwright-failed-screenshots.ts 8.36ms
 3/10 ./scripts/generate-demo-routes-file.ts 10.19ms
 4/10 ./scripts/generate-used-icons.ts 4.90ms
 5/10 ./scripts/package.json 0.87ms
 6/10 ./scripts/postbump.ts 1.56ms
 7/10 ./scripts/postchangelog.ts 3.35ms
 8/10 ./scripts/publish-canary.ts 2.51ms
 9/10 ./scripts/publish.ts 3.34ms
10/10 ./scripts/release.ts 2.27ms
CSpell: Files checked: 10, Issues found: 0 in 0 files.
```

Reply:
```
> npm run cspell
  311/5329 ./projects/addon-mobile/components/mobile-calendar-dropdown/ng-package.json 0.42ms
  312/5329 ./projects/addon-mobile/components/mobile-calendar-sheet/index.ts 0.40ms
  313/5329 ./projects/addon-mobile/components/mobile-calendar-sheet/mobile-calendar-sheet.component.ts 1.58ms
  314/5329 ./projects/addon-mobile/components/mobile-calendar-sheet/mobile-calendar-sheet.style.less 2.85ms
  315/5329 ./projects/addon-mobile/components/mobile-calendar-sheet/mobile-calendar-sheet.template.html 1.30ms
  [...]
  5328/5329 ./tsconfig.playwright.json 0.38ms
  5329/5329 ./tsconfig.spec.json 0.54ms
  CSpell: Files checked: 5329, Issues found: 0 in 0 files.
```

Reply:
No 
<img width="1245" alt="image" src="https://github.com/user-attachments/assets/a3849640-8ad7-429e-a960-1cc8ba90b0fb">

```json
"*.{js,ts,html,svg,yml,md,less,json}": [
            "prettier --write",
            "cspell"
]
```

That's mean

```
npx prettier --write ./projects/addon-mobile/components/mobile-calendar-sheet/mobile-calendar-sheet.component.ts

npx cspell ./projects/addon-mobile/components/mobile-calendar-sheet/mobile-calendar-sheet.component.ts
```

Reply:
My mistake, sorry!